XSLT/rhodes_dspace_xoai_to_mods_farnsworth.xsl

```diff
@@ -0,0 +1,327 @@
+            <xsl:choose>
+                <xsl:when test='element[@name="dc"]/element[@name="relation"]/element[@name="ispartof"]/element[@name="none"]/field[@name="value"]'>
+                    <xsl:apply-templates select='element[@name="dc"]/element[@name="relation"]/element[@name="ispartof"]/element[@name="none"]/field[@name="value"]'/>
+                </xsl:when>
+            </xsl:choose>
+       
+            <!-- rights-->
+            <xsl:choose>
+                <xsl:when test="element[@name = 'dc']/element[@name = 'rights']/element/field">
+                    <xsl:apply-templates select="element[@name = 'dc']/element[@name = 'rights']/element/field"/>                   
+                </xsl:when>
+                <xsl:otherwise>
+                    <accessCondition type="local rights statement">Rhodes College owns the rights to the digital objects in this collection. Objects are made available for educational use only and may not be used for any non-educational or commercial purpose. Approved educational uses include private research and scholarship, teaching, and student projects. For additional information please contact archives@rhodes.edu. Fees may apply</accessCondition>
+                </xsl:otherwise>
+            </xsl:choose>
+        </mods>
+    </xsl:template>
+    
+    <!-- identifiers -->
+    <xsl:template match='element[@name="dc"]/element[@name="identifier"]/element[@name="other"]/element[@name="none"]/field[@name="value"]'>
+        <identifier type="local"><xsl:apply-templates/></identifier>
+    </xsl:template>
+    
+    <!-- title -->
+    <xsl:template match="element[@name = 'dc']/element[@name = 'title']/element/field">
+        <titleInfo>
+            <title>
+                <xsl:apply-templates/>
+            </title>
+        </titleInfo>
+    </xsl:template>
+    
+    <!-- alternative title -->
+    <xsl:template match='element[@name="dc"]/element[@name="title"]/element[@name="alternative"]/element[@name="en_US"]/field'>
+        <titleInfo type="alternative">
+            <title>
+                <xsl:apply-templates/>
+            </title>
+        </titleInfo>
+    </xsl:template>
+    
+    <!-- Abstract -->
+    <xsl:template match='element[@name="dc"]/element[@name="description"]/element[@name="abstract"]/element[@name="en_US"]/field[@name="value"]'>
+        <abstract><xsl:apply-templates/></abstract>
+    </xsl:template>
+    
+    <!-- Thumbnail -->
+    <xsl:template match='element[@name="bundles"]/element[@name="bundle"][field[@name="name"][text()="THUMBNAIL"]]/element[@name="bitstreams"]/element[@name="bitstream"][1]/field[@name="url"]'>
+        <location>
+            <url access="preview"><xsl:apply-templates/></url>
+        </location>
+    </xsl:template>
+    
+    <!-- Handle -->
+    <xsl:template
+        match='element[@name = "dc"]/element[@name = "identifier"]/element[@name = "uri"]/element[@name = "none"]/field[@name = "value"]'>
+        <location>
```

Comment:
Drop those location tags and put them in the main match to limit the number you include.